third-party/realdds/src/dds-device-impl.h

```diff
@@ -51,7 +51,7 @@ class dds_device::impl
     std::queue< nlohmann::json > _option_response_queue;
 
     std::shared_ptr< dds_topic_reader > _notifications_reader;
```

Comment:
Why are you not handling notifications in a thread?
There are not a lot of traffic but we parse json and do some other jobs that can take time

Reply:
I can change, though it may have side-effects -- I'd rather do separately.

Reply:
Actually, let me try

Reply:
OK. We should monitor CPU usage and option setting delay with high resolution streaming and fix if needed.